homework/03LessonThree.js

```diff
@@ -89,18 +89,24 @@ var problemSevenAnswer;
 
 // Problem 8:
 // Create an empty array
-var emptyArray;
+var emptyArray = [
+  ['']['']
+];
 
 // Problem 9:
 // Create an array with 5 elements in it
-var lengthFiveArray;
+var lengthFiveArray = ['one', 'two', 'three', 'four', 'five'];
 
 // DO NOT MODIFY
 var nameArray = ['bob', 'fred', 'susan'];
 
 // Problem 10:
 // Replace the value 'fred' in nameArray with 'george'
 
+nameArray [1] = 'George';
```

Comment:
the bracket here should be flush with the array variable:

Ex; array[1] = 'George';